babel/core.py

```diff
@@ -60,6 +60,9 @@ def get_global(key):
             fileobj.close()
     return _global_data.get(key, {})
 
+# FIXME: Very ugly hack in order to break cyclic import dependency
+localedata.get_global = get_global
```

Comment:
Yes, there is an architectural issue.  But @erickwilder is right anyway, using an import inside `babel.localedata.load` is less magic and works just fine for this case.  I'll push a correction right away.